scripts/install/run.sh

```diff
@@ -215,74 +205,74 @@ function local_install {
     echo "Finished: npm @spartacus:registry set back to https://registry.npmjs.org/"
 }
 
-function prestart_csr {
-    if [ -z "$CSR_PORT" ]; then
-        echo "Skipping prestart csr script"
+function build_csr {
+    if [ -z "${CSR_PORT}" ]; then
+        echo "Skipping csr app build"
     else
-        printh "Prestart setup for csr app"
-        ( cd ${INSTALLATION_DIR}/csr && yarn build --prod )
+        printh "Building csr app"
+        ( cd ${INSTALLATION_DIR}/spartacus && yarn build --prod )
```

Comment:
Can we get rid of this hardcoded app name and use param instead?

Reply:
We are not passing any parameters to this function. Can you clarify?